### PR TITLE
Pupa conversion of WA

### PR DESCRIPTION
**Pull request by hiteshgarg14**
Related to #1442

-  [x] people.py
-  [x] bills.py
-  [x] events.py
-  [x] committees.py

### CODE REVIEW

**Comment by hiteshgarg14** [comment]
@jamesturk @mileswwatkins @jmcarp 
Pupa conversion is done. please review it when you get a chance :smiley:

File: billy_metadata/wa.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+settings = dict(SCRAPELIB_TIMEOUT=300)
```
Comment (jmcarp) [inline billy_metadata/wa.py]: I think this belongs in `openstates/wa/__init__.py`.

File: flake8.sh [hunk 2]
```diff
@@ -16,5 +16,7 @@ python3 -m flake8 \
     openstates/oh \
     openstates/pa \
     openstates/va \
+    openstates/wa \
     openstates/utils \
-    pupa2billy
```
Comment (jmcarp) [inline flake8.sh]: Did you want to delete this line?

File: openstates/wa/__init__.py [hunk 3]
```diff
@@ -1,75 +1,92 @@
...
+        upper_title = "Senator"
+
+        legislature = Organization(name=legislature_name,
+                                   classification="legislature")
+        upper = Organization(upper_chamber_name, classification='upper',
+                             parent_id=legislature._id)
+        lower = Organization(lower_chamber_name, classification='lower',
+                             parent_id=legislature._id)
 
-def session_list():
```
Comment (jmcarp) [inline openstates/wa/__init__.py]: This function should be kept as the `get_session_list` method.

File: openstates/wa/actions.py [hunk 4]
```diff
@@ -55,36 +56,40 @@
...
-#    Rule(r"(?i)"),
-#    Rule(r"(?i)"),
+    Rule(r"(?i)(?P<from_committee>.*) relieved of further consideration. On motion, referred to \
+         (?P<committees>.*)", 'referral-committee'),
+    Rule(r"(?i)Governor partially vetoed", 'executive-veto-line-item'),
+    Rule(r"(?i)Governor vetoed", 'executive-veto'),
+    Rule(r"(?i)Governor signed", 'executive-signature'),
+    Rule(r"(?i)Passed final passage;", 'passage'),
+    Rule(r"(?i)Failed final passage;", 'failure'),
+    # Rule(r"(?i)"),
```
Comment (jmcarp) [inline openstates/wa/actions.py]: I see these lines were here already, but it looks like we can delete them now.

File: openstates/wa/committees.py [hunk 5]
```diff
@@ -23,23 +29,26 @@ def scrape(self, chamber, term):
                 continue
 
             name = xpath(comm, "string(wa:Name)")
-            comm_id = xpath(comm, "string(wa:Id)")
+            # comm_id = xpath(comm, "string(wa:Id)")
             # acronym = xpath(comm, "string(wa:Acronym)")
-            phone = xpath(comm, "string(wa:Phone)")
+            # phone = xpath(comm, "string(wa:Phone)")
```
Comment (jmcarp) [inline openstates/wa/committees.py]: I'm wondering if we can keep some of these attributes in `extras`, or maybe add them to OCD/pupa. WDYT @jamesturk ?

**Comment by hiteshgarg14** [comment]
Going to compare data scraped tonight.....

**Comment by hiteshgarg14** [comment]
Data Comparision:-

Bills.py
- Billy :- `imported 2272 bill files` 
- Pupa:- `imported 1289 vote files` and `imported 2309 bill files`
- Difference in `related_entities` in `actions`
- Other differences are minor.

People.py 
- Good to go :+1: 

Committees.py
- Office Phone Numbers are missing because of
 ```
-    phone = xpath(comm, "string(wa:Phone)")
+  # phone = xpath(comm, "string(wa:Phone)")
```

Events.py
- The scraper was not working in the master branch.  take a  look at [Link](http://app.leg.wa.gov/mobile/MeetingSchedules/Committees?AgencyId=3&StartDate=05/09/2017&EndDate=06/08/2017&ScheduleType=2`)

**Comment by jamesturk** [comment]
thanks for the review on this one, I'm looking it over now too

**Pull request merged**